xarray/tests/test_dataarray.py

```diff
@@ -37,6 +37,12 @@ def setup(self):
                                                  names=('level_1', 'level_2'))
         self.mda = DataArray([0, 1, 2, 3], coords={'x': self.mindex}, dims='x')
 
+        self.mindex_long = pd.MultiIndex.from_product(
```

Comment:
Would you mind only defining these variables in the `test_repr_multiindex_long` method? 

We try to only define variables in `setup()` that are used in multiple methods (and even then, it's often better to just repeat setup code)

Reply:
Sure enough, was my initial thought as well, but since `self.{mindex,mda}` were only used once and defined in the fixture, i had arranged it like that.

Reply:
Done